src/hypothesis/strategies.py

```diff
@@ -630,8 +630,17 @@ def streaming(elements):
     The result is iterable (the iterator will never terminate) and
     indexable.
 
+    This function is deprecated and will be removed in a later version of
+    Hypothesis. Use data() instead.
```

Comment:
Use Sphinx cross-references, for the sake of hyperlinks:

    :func: `~hypothesis.strategies.data`